api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ListOfPatientsCurrentlyOnArtWithoutTbScreeningCohortQueries.java

```diff
@@ -0,0 +1,580 @@
+   * Denominator) during the 6 months’ period before the reporting end date
+   *
+   * </blockquote>
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition getPatientsCurrentlyOnArtWithoutTbScreening() {
+
+    CompositionCohortDefinition composition = new CompositionCohortDefinition();
+    addParameters(composition);
+    composition.setName("Currently on ART without TB Screening");
+
+    CohortDefinition txCurr = txCurrCohortQueries.getTxCurrBaseCohort();
+    CohortDefinition txTbDenominator = txtbCohortQueries.getDenominator();
+
+    composition.addSearch(
+        "tx-curr", EptsReportUtils.map(txCurr, "endDate=${endDate},location=${location}"));
+    composition.addSearch(
+        "txtb-denominator",
+        EptsReportUtils.map(
+            txTbDenominator, "startDate=${endDate-6m},endDate=${endDate},location=${location}"));
+
+    composition.setCompositionString("tx-curr AND NOT txtb-denominator");
+
+    return composition;
+  }
+
+  /**
+   * <b>Technical Specs</b>
+   *
+   * <blockquote>
+   *
+   * <p><b>Total number of patients on the List (TB_NSCRN_FR2) </b>
+   *
+   * <p>Number of Patients Currently on ART without TB screening and at least one Clinical
+   * Consultation in last 6 months
+   *
+   * </blockquote>
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition
+      getPatientsCurrentlyOnArtWithoutTbScreeningAndWithClinicalConsultationInLast6Months() {
+
+    CompositionCohortDefinition composition = new CompositionCohortDefinition();
+    addParameters(composition);
+    composition.setName("Currently on ART without TB Screening and With Clinical Consultation");
+
+    CohortDefinition currentlyOnArtWithoutTbScreening =
+        getPatientsCurrentlyOnArtWithoutTbScreening();
+    CohortDefinition withClinicalConsultationInLast6Months =
+        getPatientsWithClinicalConsultationInLast6Months();
+
+    composition.addSearch(
+        "onArtWithoutScreening",
+        EptsReportUtils.map(
+            currentlyOnArtWithoutTbScreening, "endDate=${endDate},location=${location}"));
+    composition.addSearch(
+        "withConsultation",
+        EptsReportUtils.map(
+            withClinicalConsultationInLast6Months,
+            "startDate=${endDate-6m},endDate=${endDate},location=${location}"));
+
+    composition.setCompositionString("onArtWithoutScreening AND withConsultation");
+
+    return composition;
+  }
+
+  /**
+   * <b>Technical Specs</b>
+   *
+   * <blockquote>
+   *
+   * <p><b>Total number of patients on the List (TB_NSCRN_FR2) </b>
+   *
+   * <p>Number of Patients Currently on ART without TB screening and at least one Clinical
+   * Consultation in last 6 months
+   *
+   * </blockquote>
+   *
+   * @return {@link CohortDefinition}
+   */
+  public CohortDefinition
+      getPatientsCurrentlyOnArtWithoutTbScreeningAndWithoutClinicalConsultationInLast6Months() {
```

Comment:
Looks like this method is NOT referenced anywhere

Reply:
Fixed